plugins/stt/pocketsphinx-stt/sphinxvocab.py

```diff
@@ -10,7 +10,7 @@
 
 
 def delete_temp_file(file_to_delete):
-    if False:
+    if True:
```

Comment:
You are right, it is only a placeholder. It is helpful for me to sometimes turn off the automatic cleanup of temporary files. Then I can follow the logic through them and figure out why the next step fails exactly. That was the whole purpose of the "delete_temp_file" function, actually. I accidentally left it set to False in an earlier pull request but nobody caught it.

What I would like to do with this is connect it to a command line variable and possibly also a profile setting, so a developer can turn on and off deleting temp files. Since this is really only useful when you also have --debug turned on, I might require both that Naomi is running in debug mode and that a switch has been set to keep temporary files.

Reply:
Fair enough. I like that plan!